src/lib/Scenes/City/CitySavedList.tsx

```diff
@@ -65,7 +65,7 @@ class CitySavedList extends React.Component<Props, State> {
           cityName={name}
           bucket={edges.map(e => e.node) as any}
           type="saved"
-          relay={relay}
+          relay={relay as RelayProp}
```

Comment:
This is good to know

Reply:
I wouldn’t bother remembering this too much, the passing around of the relay prop that is being done here should really be replaced with a `SystemContext`, like in Reaction–also because in reality it probably only needs the env for a mutation.

Reply:
Right, that makes sense, we had talked about using the context API to pass relay down instead here.